storage/tests/unit/test_blob.py

```diff
@@ -366,7 +366,7 @@ def test__make_transport(self, fake_session_factory):
 
     def test__get_download_url_with_media_link(self):
         blob_name = 'something.txt'
-        bucket = mock.Mock(spec=[])
+        bucket = _Bucket(name='IRRELEVANT')
```

Comment:
I would greatly prefer `mock.Mock`.  It confines to the test what the expectation is, using a format that contributors are going to be familiar with. Having to look up `_Bucket` should not be necessary.